### PR TITLE
Feature/637 make user dropdown searchable but without js for robin :)

### PR DESCRIPTION
**Pull request by ManuelMoeri**

### CODE REVIEW

File: app/javascript/application.js [hunk 1]
```diff
@@ -1,2 +1,4 @@
 import "@hotwired/turbo-rails"
 import "./controllers"
+import "choices.js"
+
```
Comment (RandomTannenbaum) [inline app/javascript/application.js:4]: Is this import necessary? :eyes:

File: app/views/people/_search.html.haml [hunk 2]
```diff
@@ -2,7 +2,7 @@
   %div.w-50
     = form_with do |f|
       %section{"data-controller"=>"dropdown"}
-        = f.collection_select :person_id, Person.all.sort_by(&:name), :id, :name,  select_when_availabale(person) , {class: "form-select w-100", "data-action": "change->dropdown#handleChange", "data-value": "/people/"}
+        = f.collection_select :person_id, Person.all.sort_by(&:name.downcase), :id, :name, select_when_availabale(person), {id: "dropdown", class: "form-select w-100", "data-action": "change->dropdown#handleChange", "data-value": "/people/"}
   %a.d-flex.justify-content-between#new-person-button{href: "/people/new"}
     %img.pe-1{src: "/assets/plus-lg.svg"}
-    Neues Profil
\ No newline at end of file
+    Neues Profile
```
Comment (RandomTannenbaum) [inline app/views/people/_search.html.haml:8]: :england:

File: spec/features/people_spec.rb [hunk 3]
```diff
@@ -7,35 +7,62 @@
       sign_in auth_users(:user), scope: :auth_user
     end
 
-    let(:list) {Person.all.sort_by(&:name) }
+    let(:list) {Person.all}
```
Comment (RandomTannenbaum) [inline spec/features/people_spec.rb:11]: Maybe rename to person_list or similar

**Review by RandomTannenbaum — COMMENTED** [review]
:rocket:

File: spec/features/people_spec.rb (hunk 3 above)
Comment (ManuelMoeri) [reply spec/features/people_spec.rb:11]: Done.

**Review by Robin481 — CHANGES_REQUESTED** [review]
- Fix spacing on root page
- Fix dropdown being disabled when returning to it via browser back
- Remove too many seeded people

Besides that lovely PR :rock:

**Review by Robin481 — APPROVED** [review]

**Pull request closed without merging**